File: php/examples/figuras/php/triangle.php
<?php
class triangle extends  FigureGeometric{

    private float $base;
    private float $height;

    public function __construct(float $base, float $height){
        parent:: __construct("Triangle","Triangulo");
            $this->base = $base;
            $this->height= $height;
        }
            public function calculateArea(): float{
                return ($this->base*$this->height)/2;
            }       
        }
?> 

File: java/figuras/src/main/java/com/maicol/figuras/triangle.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package com.maicol.figuras;

/**
 *
 * @author Maicol
 */
public class triangle extends figureGeometric{
    private double base;
    private double height;

    public triangle(double base, double height) {
        super("Triangle", "Triángulo");
        this.base = base;
        this.height = height;
    }

    @Override
    public double calculateArea() {
        return (base * height) / 2;
    }
}